src/core_plugins/kibana/public/visualize/editor/editor.html

```diff
@@ -80,6 +80,9 @@
     <div class="vis-editor-canvas" ng-class="{ embedded: !chrome.getVisible() }">
       <visualize
         vis="vis"
+        reporting-item
```

Comment:
Agreed, if we want this to be part of the standardized visualization API I think it makes total sense for this to become more general.